BParkingNano/python/BToKLL_cff.py

```diff
@@ -99,6 +99,8 @@
         kIdx = uint('k_idx'),
         minDR = ufloat('min_dr'),
         maxDR = ufloat('max_dr'),
+        asym_transverse_momentum = ufloat("asym_transverse_momentum"),
+        Longitudinal_distance_dilepton_kaon = ufloat("kaon_dilepton_longitudinal_distance"),
```

Comment:
better shorthand name? dz_ll_k? and then add a "doc" comment with a more complete description?